src/Objs/_metadataEditingConfig.js

```diff
@@ -25,4 +25,44 @@ export const metadataPropertiesGroup = {
 export const socialCardsPropertiesGroup = {
   title: "Social cards",
   component: "SocialCardsTab",
+  properties: ["tcCreator", "tcDescription", "ogDescription"],
 };
+
+export const socialCardsValidations = [
+  [
+    "tcCreator",
+
+    tcCreator => {
+      if (tcCreator && tcCreator.charAt(0) !== "@") {
+        return {
+          message: "The username should start with @.",
```

Comment:
See discussion on https://github.com/Scrivito/scrivito_example_app_js/pull/265#discussion_r360316975.

```suggestion
          message: "The creator should start with @.",
```